Event Logo cut-off with long event names fixed

Ruchip16 opened the pull request:
<!-- 
(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
-->

<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->

Fixes #8029 

#### Short description of what this resolves:

Event logo with long names used to get cut-off while we scroll down the page

#### Changes proposed in this pull request:

Gave some styling so that text would appear like this

![image](https://user-images.githubusercontent.com/72685035/144700236-3d916396-9ace-46c8-97dc-5fa29cf1c99e.png)


#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes <!-- use `ember test` to run all the tests -->
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

maze-runnar commented:
@Ruchip16 not working for mobile view - 
![Screenshot from 2021-12-05 12-41-15](https://user-images.githubusercontent.com/56407566/144737294-e268738e-514b-4a28-94d9-381e4ac1fb3e.png)

Ruchip16 commented:
> @Ruchip16 not working for mobile view - ![Screenshot from 2021-12-05 12-41-15](https://user-images.githubusercontent.com/56407566/144737294-e268738e-514b-4a28-94d9-381e4ac1fb3e.png)

yaa, coz I have only changed for web app and not for mobile view coz for that particular example it was working fine for mobile view hence did not pay much attention there

pc-beast reviewed (changes requested):
Please make necessary changes for the mobile view too. Also please use helper classes for margins from ``_utils.scss``` If defining a class is a necessity then define it there.

pc-beast commented:
@Ruchip16 updates?

Ruchip16 commented:
> @Ruchip16 updates?

ya, I haven't made the changes yet but will probably make them tonight and would push the commit here

Ruchip16 commented:
> Please make necessary changes for the mobile view too. Also please use helper classes for margins from ``_utils.scss``` If defining a class is a necessity then define it there.

where can I find the media queries for the particular class coz I am not able to find them in the styles folder

Ruchip16 commented:
> > Please make necessary changes for the mobile view too. Also please use helper classes for margins from ``_utils.scss``` If defining a class is a necessity then define it there.
> 
> where can I find the media queries for the particular class coz I am not able to find them in the styles folder @pc-beast



> Please make necessary changes for the mobile view too. Also please use helper classes for margins from ``_utils.scss``` If defining a class is a necessity then define it there.

Also I have added margin for the ```mr-8 class``` which was defined already in ```utils.scss``` but if I have to add margin or padding to the main div class ```ui info d-flex``` or sub headings class like ```event time ``` , ```event name text-white``` , ```event location``` will it be fine if I define it there or like it has to be defined elsewhere/ already defined elsewhere or just inline CSS would work like I have added it already because I went through entire styles folder in ```app/styles``` dir but I couldn't find there

pc-beast commented:
just define classes if you are not able to find one according to your need. Many of the classes used are from SemanticUI as we are using it. Please see if can be solved with classes from semantic UI. If not define your own classes in related files. Try to complete it asap as it is a small issue and it should not become a blocker.

Ruchip16 commented:
> just define classes if you are not able to find one according to your need. Many of the classes used are from SemanticUI as we are using it. Please see if can be solved with classes from semantic UI. If not define your own classes in related files. Try to complete it asap as it is a small issue and it should not become a blocker.

yaa okay

pc-beast commented on app/styles/partials/utils.scss:
```diff
@@ -236,4 +236,17 @@
 
 .mr-8 {
   margin-right: 2rem !important;
+  margin-bottom: -120px;
```
mr here means margin-right..do not change anything in existing classes.

Ruchip16 replied:
okay will revert this

pc-beast commented on app/templates/public.hbs:
```diff
@@ -7,10 +7,10 @@
             </LinkTo>
           {{/if}}
-          <div>
+          <div style="margin-left:220px">
```
why inline?

Ruchip16 replied:
Because I knew that those classes are of semantic UI and I did not get any file or folder where the styling of these classes was present hence I did not remove the inline CSS from here

pc-beast commented on app/styles/partials/utils.scss:
```diff
@@ -236,4 +236,17 @@
+  margin-bottom: -120px;
+}
+
+.event.time {
+  margin-left: 220px;
+}
+
+.event.name {
+  margin-left: 220px;
+}
+
+.event.location {
+  margin-left: 220px;
```
these are classes from SemanticUI, do not add properties here

Ruchip16 replied:
ya hence i asked u that where would i get this styling

Ruchip16 replied:
> ya hence i asked u that where would i get the styling of these classes

progmatic-99 reviewed (commented):
@Ruchip16 Please upload a screenshot of your changes in both pc and mobile screen size.

Ruchip16 commented:
> @Ruchip16 Please upload a screenshot of your changes in both pc and mobile screen size.

![image](https://user-images.githubusercontent.com/72685035/145939810-5c54f2e6-3e53-40a5-9fb4-e0f161ec229b.png)

progmatic-99 commented:
@mariobehling pls review once

pc-beast commented on app/styles/partials/utils.scss:
```diff
@@ -159,7 +159,7 @@
 
 .mb-8 {
-  margin-bottom: 2rem !important;
+  margin-bottom: -7.5rem !important;
```
Do not change this definition

pc-beast reviewed (commented):
Why have you added a module in file changes?

Ruchip16 commented:
> Closing due to inactivity.

🥺🥺🥺🥺

pc-beast commented:
Feel free to reopen and finish it if you are still working on it.

Ruchip16 commented:
> Feel free to reopen and finish it if you are still working on it.

yeah I will finish it soon, i was stunned for a minute though

pc-beast commented:
Try to finish it if you are still working on it. Updated the latest changes here. Thanks.

Ruchip16 commented:
> Try to finish it if you are still working on it. Updated the latest changes here. Thanks.

yeahh, actually past week I had some interviews hence did not get time to work on this but will definitely do it soon, sorry for the delay

Ruchip16 commented on app/templates/public.hbs:
```diff
@@ -7,10 +7,10 @@
           {{#if this.model.logoUrl}}
             <LinkTo @route="public">
-              <img src="{{this.model.logoUrl}}" class="logo mr-8" />
+              <img src="{{this.model.logoUrl}}" class="logo mr-8" style="margin-bottom: -120px;"/>
```
I added inline CSS here coz I couldn't create just another class for the logo as there is one already and plus I had to do changes in margin-bottom and not margin-right

progmatic-99 reviewed (changes requested):
Please always upload a ss when you are doing changes to the UI.
Thanks

Ruchip16 commented:
> Existing events do not have the same UI as before.
> 
> **Please test the changes you make with creating a dummy event that tests corner cases and also make sure that existing events stay the same.** Thanks ![Screenshot 2021-12-22 at 18-57-44 OSBA Members Products eventyay](https://user-images.githubusercontent.com/42090582/147100080-1f8071b9-73b4-4e0c-93e3-923a6675924b.png)

yaa I noticed that like entire UI gets affected due to that changes but then only inline CSS would prolly help coz otherwise if I create a class like this and do the changes like that it would get changed entirely how should I overcome this?

pc-beast commented:
> yaa I noticed that like entire UI gets affected due to that changes but then only inline CSS would prolly help coz otherwise if I create a class like this and do the changes like that it would get changed entirely how should I overcome this?

You know you can define a class for everything you do in inline styling, right? Please understand that If we guide you with every word in code, it is better that we implement it ourselves.
